manager/runner.go

```diff
@@ -513,6 +534,12 @@ func (r *Runner) Run() error {
 				// Record that at least one template was rendered.
 				renderedAny = true
 
+				// Send the signal that the template got rendered
+				select {
+				case r.renderedCh <- tmpl.Path:
+				default:
```

Comment:
Hmm yea - I'm just worried that there's a race where the channel is full and the listener never receives that message, even though it's reading from the channel.